trying to install on ubuntu 19.10 error `make: *** No targets specified and no makefile found.  Stop`

Hi.

Im trying to install on ubuntu 19.10, the steps I made are as follow:

 `git clone git@github.com:kitone/subtitleeditor.git`
`cd subtitleeditor/`
`./autogen.sh`
`make`

running make gives me the following error:

`make: *** No targets specified and no makefile found.  Stop`

--- a/README.md
+++ b/README.md
@@ -64,6 +64,7 @@ Source tarballs, installing from a source tarball:
 
 ```bash
 ./autogen.sh
+./configure
 make
 sudo make install
 ```

